### PR TITLE
New Textures + Reorganizing Assets

### PR DESCRIPTION
**Pull request by fnnbrr**
NOTE: Modified NursePatrol.cs so that you now give it ONE GameObject containing multiple child "waypoints" (Transforms).  This replaces the old system of dragging all of the individual waypoints onto each nurse in the inspector.

### CODE REVIEW

File: Assets/Scripts/NursePatrol.cs [hunk 1]
```diff
@@ -5,8 +5,9 @@
 
 public class NursePatrol : MonoBehaviour
 {
-    public Transform[] points;
+    public GameObject Patrol_Waypoints;
```
Comment (jurcaua) [inline Assets/Scripts/NursePatrol.cs]: I think this can just be type `Transform`.

**Review by jurcaua — COMMENTED** [review]
Looks good outside of my small comment!

**Comment by justineechen** [comment]
It looks like some of the meta files were pushed so there's 119 file changes, I think we should leave the .meta files out but idk why the git ignore didn't pick it up

**Comment by jurcaua** [comment]
I think .meta files should be good to keep in git actually!

**Comment by justineechen** [comment]
Okay! Ignore my comment finnbarr 🤡

File: Assets/Scripts/NursePatrol.cs (hunk 1 above)
Comment (fnnbrr) [reply Assets/Scripts/NursePatrol.cs]: Fixed

**Pull request merged**